extensions/logging/src/decorators/log.decorator.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/extension-logging
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {intercept} from '@loopback/context';
+import {LoggingBindings} from '../keys';
+
+/**
+ * @log decorator for method invocations.
+ *
+ * @example
+ * ```ts
+ * import {log} from '@loopback/extension-logging';
+ *
+ * export class HelloController {
+ *   @log()
+ *   hello(name: string) {
+ *     return `Hello, ${name}`;
+ *   }
+ * }
+ * ```
+ */
+export function log() {
+  // A shortcut to `@intercept` that invokes the winston interceptor that logs
+  // method invocations
+  return intercept(LoggingBindings.WINSTON_INTERCEPTOR);
```

Comment:
Please find a name that better communicates what is this interceptor for. For example: `WINSTON_INVOCATION_ARGS_LOGGING_INTERCEPTOR`.

Reply:
`WINSTON_INVOCATION_LOGGER`